cmd/jobobject-util/main.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/Microsoft/hcsshim/internal/jobobject"
+	"github.com/urfave/cli"
+)
+
+const (
+	cpuLimitFlag    = "cpu-limit"
+	cpuWeightFlag   = "cpu-weight"
+	memoryLimitFlag = "memory-limit"
+	affinityFlag    = "cpu-affinity"
+
+	usage = `jobobject-util is a command line tool for getting and setting job object limits`
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "jobobject-util"
+	app.Commands = []cli.Command{
+		getJobObjectLimitsCommand,
+		setJobObjectLimitsCommand,
+	}
+	app.Usage = usage
+
+	if err := app.Run(os.Args); err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+}
+
+var getJobObjectLimitsCommand = cli.Command{
+	Name:      "get",
+	Usage:     "gets the job object's resource limits",
+	ArgsUsage: "get [flags] <name>",
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  cpuLimitFlag,
+			Usage: "Optional: get job object's CPU limit",
+		},
+		cli.BoolFlag{
+			Name:  cpuWeightFlag,
+			Usage: "Optional: get job object's CPU weight.",
+		},
+		cli.BoolFlag{
+			Name:  memoryLimitFlag,
+			Usage: "Optional: get job object's memory limit in bytes.",
+		},
+		cli.BoolFlag{
+			Name:  affinityFlag,
+			Usage: "Optional: get job object's CPU affinity as a bitmask.",
+		},
+	},
+	Action: func(cli *cli.Context) error {
+		ctx := context.Background()
+		name := cli.Args().First()
+		if name == "" {
+			return errors.New("`get` command must specify a target job object name")
+		}
+		options := &jobobject.OptionsJobObject{
+			Name:          name,
+			Notifications: false,
+			UseNTVariant:  true,
```

Comment:
Should this just be configurable? We don't use the nt variant to make the job for job containers